src/locales/en.json

```diff
@@ -96,6 +96,10 @@
     "title": "Games"
   },
   "games": {
+    "quizgame": {
+      "title": "Quiz Game",
+      "screenshot": "https://firebasestorage.googleapis.com/v0/b/capstone-istanbul-juniper.appspot.com/o/QuizGame%2Fscreenshot.png?alt=media&token=d2736b79-3595-483a-be77-1afce5ddd4a7"
```

Comment:
notice how the 3 are duplicated = already a warning

Reply:
object removed